tutorials/scripting/signals_concise_example.rst

```diff
@@ -0,0 +1,113 @@
+.. _doc_signals_concise_example:
+
+Signals, concise example
+========================
+
+Signals are easy.
+
+Make Signal (in sender):
+
+``signal SIGNALNAME``
+
+
+Emit Signal (in sender):
+
+``SIGNALNAME.emit(arg1, arg2, ...)``
+
+
+Connect Signal (in receivers):
+
+``sender.SIGNALNAME.connect(FUNCTIONNAME)``
+
+
+Make functions (in receivers):
+
+``func FUNCTIONNAME(arg1, arg2, ...):``
```

Comment:
All of this is GDScript. You should use `code-tab` to provide the C# equivalent.

I'm a bit worried about using all caps names since users may misunderstand this as the naming convention. I'm also hesitant to provide examples with invalid syntax (such as ellipsis). So I refrained from doing so in the C# examples.

```suggestion
Make Signal (in sender):

.. tabs::
 .. code-tab:: gdscript GDScript

    signal SIGNALNAME

 .. code-tab:: csharp

     [Signal]
     public delegate void SignalNameEventHandler();

Emit Signal (in sender):

.. tabs::
 .. code-tab:: gdscript GDScript

    SIGNALNAME.emit(arg1, arg2, ...)

 .. code-tab:: csharp

     EmitSignal(SignalName.SignalName, arg1, arg2);

Connect Signal (in receivers):

.. tabs::
 .. code-tab:: gdscript GDScript

    sender.SIGNALNAME.connect(FUNCTIONNAME)

 .. code-tab:: csharp

    sender.SignalName += FunctionName;

Make functions (in receivers):

.. tabs::
 .. code-tab:: gdscript GDScript

    func FUNCTIONNAME(arg1, arg2, ...):

 .. code-tab:: csharp

    private void FunctionName(int arg1, int arg2)
```